Documentation/En/Index.xyl

```diff
@@ -111,6 +111,9 @@ echo $bench;
  *     __global__  ||||||||||||||||||||||||||||||||||||||||||||||||||||    45ms, 100.0%
  *     aMark       |||||||||||||                                           11ms,  24.8%
  */</code></pre>
+  <p>We can also pause all marks and get what marks were running with the
```

Comment:
`what` => `quoi`
`which` => `quel(le)`

it seems more natural to use "which" although  they seem to have the same meaning... As @Hywan you choose :p